project-keeper/src/test/java/com/exasol/projectkeeper/mavenrepo/LatestVersionTest.java

```diff
@@ -0,0 +1,103 @@
+package com.exasol.projectkeeper.mavenrepo;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.IOException;
+import java.io.StringReader;
+import java.net.MalformedURLException;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import com.exasol.projectkeeper.mavenrepo.MavenRepository.JsonContentException;
+import com.exasol.projectkeeper.mavenrepo.Version.UnsupportedVersionFormatException;
+
+import jakarta.json.*;
+
+//[utest->dsn~verify-own-version~1]
+class LatestVersionTest {
```

Comment:
- Split into MavenRepositoryTest & VersionTest
- Add tests for hashcode & equals, see https://jqno.nl/equalsverifier/

Reply:
Uh Oh: @EqualsAndHashCode and nl.jqno.equalsverifier.EqualsVerifier ... 
discussion https://github.com/projectlombok/lombok/issues/1872  ended in `suppress(Warning.STRICT_INHERITANCE)` 😦